src/relayclient/types/Environments.ts

```diff
@@ -8,24 +8,28 @@ class Environment {
   public readonly gtxdatanonzero: number
   public readonly gtxdatazero: number
   public readonly chainId: number
+  public readonly mintxgascost: number
 
   constructor (env: Environment) {
     this.gtxdatanonzero = env.gtxdatanonzero
     this.gtxdatazero = env.gtxdatazero
     this.chainId = env.chainId
+    this.mintxgascost = env.mintxgascost
   }
 }
 
 export const environments = {
   istanbul: new Environment({
     gtxdatanonzero: 16,
     gtxdatazero: 4,
-    chainId: 1
+    chainId: 1,
+    mintxgascost: 21000
```

Comment:
Well, I wanted to be consistent, that if we take the other parameters by fork/chain/network, then we should take that value as well from environments. Will it change? it might just like many other opcode gas costs have changed, though unlikely in near future probably.